src/pages/merge/IndividualsMergePending.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { useParams } from 'react-router';
+import { useIntl } from 'react-intl';
+
+import { useTheme } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Skeleton from '@material-ui/lab/Skeleton';
+import Typography from '@material-ui/core/Typography';
+
+import useDocumentTitle from '../../hooks/useDocumentTitle';
+import useIndividualMergeRequest from '../../models/individual/useIndividualMergeRequest';
+import BaoConfused from '../../components/svg/BaoConfused';
+import BaoSpills from '../../components/svg/BaoSpills';
+import ButtonLink from '../../components/ButtonLink';
+import CustomAlert from '../../components/Alert';
+import MainColumn from '../../components/MainColumn';
+import Text from '../../components/Text';
+import { formatDateCustom } from '../../utils/formatters';
+
+/**
+ * Because the Bao SVGs have different dimensions, these widths were selected
+ * to keep the Bao characters approximately the same size on the page.
+ */
+const baoSpillsWidth = 430;
+const baoConfusedWidth = 280;
+
+function Bao({ isError }) {
+  const theme = useTheme();
+  const themeColor = theme.palette.primary.main;
+
+  return isError ? (
+    <BaoSpills
+      themeColor={themeColor}
+      style={{ width: baoSpillsWidth, margin: '72px 8px 20px 0' }}
+    />
+  ) : (
+    <BaoConfused
```

Comment:
I don't love BaoConfused for an expected user experience. I don't think that this comment is really in the purview of this ticket, but I almost think I'd like BaoDetective better here? I dunno. @brmscheiner would it be crazy to commission a BaoWaiting?

Reply:
dont think we have time to worry about new ones right now. feel free to try baoDetective though and switch it out if it looks better